NOM_visualizations/R/NOM_R_notebook.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Can you adjust your <code>geom_histogram</code> parameters to prevent the warning message? Or use suppress warnings? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>No more warnings!</p>